LICENSE

```diff
@@ -1,4 +1,5 @@
-  Copyright 2021-present
+
+   Copyright 2021-present Crypto.org
```

Comment:
btw why don't we use cronos labs?

Reply:
cronos labs are used on the ecosystem projects of cronos labs: https://github.com/cronos-labs